build.gradle

```diff
@@ -46,7 +46,7 @@ publishing {
 repositories {
     mavenLocal()
     // Remove the commented code below once TransportService is published to maven
```

Comment:
since we're removing the commented code, can we remove the comment to remove the commented code?  (Or add a comment above the comment to remove the comment saying we should remove the comment...? Er, no...)

Reply:
^^^

Reply:
^^^ can we please delete this comment?